Source/Common/pch_common.h

```diff
@@ -52,6 +52,7 @@
 #include <codecvt>
 #include <iomanip>
 #include <functional>
+#include <list>
```

Comment:
nit: should this be alphabetical?

Reply:
sure, looks like it mostly is.  Will fix the few that aren't